ppl/src/test/java/org/opensearch/sql/ppl/antlr/PPLSyntaxParserTest.java

```diff
@@ -7,11 +7,17 @@
 package org.opensearch.sql.ppl.antlr;
 
 import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThrows;
 
 import org.antlr.v4.runtime.tree.ParseTree;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
+import org.opensearch.sql.common.antlr.SyntaxCheckException;
+
+import java.util.List;
+import java.util.stream.Stream;
```

Comment:
Reorder imports in sake of checkstyle

Reply:
Fixed all checkstyle failures in https://github.com/Bit-Quill/opensearch-project-sql/pull/283/commits/1c64d5c6400e6de10759d9934ad0117e1c8123a4